ormolu.el

```diff
@@ -42,74 +43,23 @@
   :safe #'stringp)
 
 (defcustom ormolu-extra-args nil
-  "Extra arguments to give to ormolu"
+  "Extra arguments to give to ormolu."
   :group 'ormolu
   :type 'sexp
   :safe #'listp)
 
-(defcustom ormolu-reformat-buffer-on-save nil
-  "Set to t to run `ormolu-format-buffer` when a buffer in `:mode` is saved."
-  :group 'ormolu
-  :type 'boolean
-  :safe #'booleanp)
-
-;; Minor mode
-
 (defvar ormolu-mode-map (make-sparse-keymap)
-  "Local keymap used for `ormolu-mode`.")
-
-;;;###autoload
-(define-minor-mode ormolu-mode
-  "Minor mode to format code with the \"ormolu\" program.
-
-Provide the following keybindings:
-
-\\{ormolu-mode-map}"
-  :init-value nil
-  :keymap ormolu-mode-map
-  :lighter " Or"
+  "Local keymap used for `ormolu-format-on-save-mode`.")
+
+;;;###autoload (autoload 'ormolu-format-buffer "ormolu" nil t)
+;;;###autoload (autoload 'ormolu-format-region "ormolu" nil t)
+;;;###autoload (autoload 'ormolu-format-on-save-mode "ormolu" nil t)
+(reformatter-define ormolu-format
+  :program ormolu-process-path
+  :args ormolu-extra-args
```

Comment:
For me, it doesn't work without specifying a source file to format:

```suggestion
  :args (append sqlformat-args '("/dev/stdin"))
```

Reply:
Hm, weird, works just fine for me. Will change though.